lib/client.js

```diff
@@ -15,7 +15,7 @@ var Client = function () {};
 var defaultOpt = {
   port: 30080,
   sendBounds: true,
-  verbose: false
+  verbose: 1
```

Comment:
Good point. Lets keep backward compatibility with `verbose`, but transform`verbose` to `logLevel` internally. However, make changes to the documentation in `README.md` by replacing `verbose` to `logLevel`.